resources/assets/uad_lists.json

```diff
@@ -20050,5 +20050,257 @@
     "neededBy": null,
     "labels": null,
     "removal": "Recommended"
+  },
+  {
+    "id": "com.coloros.compass2",
+    "list": "Oem",
+    "description": "Compass App",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.coloros.gamespaceui",
+    "list": "Oem",
+    "description": "GameSpace For older COS/RUI",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.coloros.onekeylockscreen",
+    "list": "Oem",
+    "description": "Put the device in lock mode when you click the app icon",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.coloros.oshare",
+    "list": "Oem",
+    "description": "Oppo Share\nUnified file sharing service",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.coloros.phonemanager",
+    "list": "Oem",
+    "description": "Phone Manager\nProvides optimization tools and various security scanning services.\nHas obviously a lot of permissions which can't be revoke.",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.coloros.securepay",
+    "list": "Oem",
+    "description": "Part of Phone Manager App\nRuns in background to provide secure environment on banking apps",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.coloros.systemclone",
+    "list": "Oem",
+    "description": "System Cloner\nCreates multiple user on device",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.coloros.video",
+    "list": "Oem",
+    "description": "Video Player",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.heytap.browser",
+    "list": "Oem",
+    "description": "Browser App\nSpam device with ads and notifications",
+    "dependencies": null,
+    "neededBy": null,
+    "labels": null,
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.heytap.habit.analysis",
+    "list": "Oem",
+    "description": "",
```

Comment:
As it got the 'Recommended' status regarding `removal`, I think it's safe to say you didn't experience issues uninstalling/disabling this package. According to the APK-name it seems some sort of analytics APK, similar to OnePlus' APKs (such as `net.oneplus.commonlogtool` and `net.oneplus.odm`, so maybe put that in the description?

Reply:
Yes, already tested on  oppo device.

Reply:
Can you include your findings in the description?

Reply:
I don't know what exactly this app does because it's not available in my device so I can't test.
The only thing I know is it's name is IntelligentAnalyticsSystem.

Reply:
But you said you tested it on an OPPO device, no? Or do you mean you tested the APK by removing it, didn't notice differences, and thus don't know what it does?

Reply:
Got it. Could you take a look at my pending comments?